pkg/build/api/types.go

```diff
@@ -63,6 +63,23 @@ const (
 	// forces the build to be processed by the build controller queue without waiting
 	// for a resync.
 	BuildAcceptedAnnotation = "build.openshift.io/accepted"
+
+	// BuildStartedEvent is the reason associated with the event registered when a build is started (pod is created).
```

Comment:
Should be BuildStartedEventReason in comment (also should be fixed in the rest of the comments to match constant names)